docs/deploy-rbd.md

```diff
@@ -26,22 +26,25 @@ make image-cephcsi
+| `--metricspath`          | `"/metrics"`          | Path of prometheus endpoint where metrics will be available                                                                                                                                                                                                                          |
+| `--enablegrpcmetrics`    | `false`               | [Deprecated] Enable grpc metrics collection  and start prometheus server                                                                                                                                                                                                             |
+| `--polltime`             | `"60s"`               | Time interval in between each poll                                                                                                                                                                                                                                                   |
+| `--timeout`              | `"3s"`                | Probe timeout in seconds                                                                                                                                                                                                                                                             |
+| `--histogramoption`      | `0.5,2,6`             | [Deprecated] Histogram option for grpc metrics, should be comma separated value (ex:= "0.5,2,6" where start=0.5 factor=2, count=6)                                                                                                                                                   |
+| `--domainlabels`         | _empty_               | Kubernetes node labels to use as CSI domain labels for topology aware provisioning, should be a comma separated value (ex:= "failure-domain/region,failure-domain/zone")                                                                                                             |
+| `--rbdhardmaxclonedepth` | `8`                   | Hard limit for maximum number of nested volume clones that are taken before a flatten occurs                                                                                                                                                                                         |
+| `--rbdsoftmaxclonedepth` | `4`                   | Soft limit for maximum number of nested volume clones that are taken before a flatten occurs                                                                                                                                                                                         |
+| `--skipforceflatten`     | `false`               | skip image flattening on kernel < 5.2 which support mapping of rbd images which has the deep-flatten feature                                                                                                                                                                         |
```

Comment:
I thought we were killing this in favor of just force-flattening an image upon first use on an incompatible node?

Reply:
@dillaman This is just an option provided to the user if the kernel version which supports deep-flatten is not in the list that cephcsi maintains, the user can still skip flattening of the rbd images before use in NodeStageRequest, the user doesn't need to wait for a new cephcsi release which adds the kernel to the list of the kernel it maintains that supports the deep-flatten feature.